plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/kvm/ha/KVMHAConfig.java

```diff
@@ -53,4 +53,32 @@
     public static final ConfigKey<Long> KvmHAFenceTimeout = new ConfigKey<>("Advanced", Long.class, "kvm.ha.fence.timeout", "60",
             "The maximum length of time, in seconds, expected for a fence operation to complete.", true, ConfigKey.Scope.Cluster);
 
+    public static final ConfigKey<Integer> KvmHaWebservicePort = new ConfigKey<Integer>("Advanced", Integer.class, "kvm.ha.webservice.port", "8443",
+            "It sets the port used to communicate with the KVM HA Agent Microservice that is running on KVM nodes. Default value is 8443.",
+            true, ConfigKey.Scope.Cluster);
+
+    public static final ConfigKey<Boolean> IsKvmHaWebserviceEnabled = new ConfigKey<Boolean>("Advanced", Boolean.class, "kvm.ha.webservice.enabled", "false",
```

Comment:
@nvazquez maybe I need to rename or change the description to make it more clear.

This is to disable the KVM HA helper. It is an "on/off" switch that makes CloudStack check or not the added webserver on KVM nodes.

1. When it is set to `false` (**default**) it does not validate via the KVM HA Helper Client, thus it keeps the normal behavior for the CloudStack KVM HA flow.
2. When it is enabled, then it will check the web server's health.

Reply:
@GabrielBrascher thanks, maybe it can be simply renamed to `kvm.ha.webservice.check.enabled` or similar?